corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1,6 +1,8 @@
 /*global Marionette */
 
-hqDefine("cloudcare/js/formplayer/menus/views", function () {
+hqDefine("cloudcare/js/formplayer/menus/views", [
+    'analytix/js/kissmetrix',
```

Comment:
Looks like importing here is explicitly disallowed. I'l have to find somewhere downstream to put these trackers.

Reply:
yeah, this isn't allowed because web apps doesn't use requirejs (yet). The way to do this is use `hqImport` in the body: https://commcare-hq.readthedocs.io/js-guide/dependencies.html#non-requirejs